dataUtil/constants.py

```diff
@@ -0,0 +1,2 @@
+MS = 1000
```

Comment:
units? might consider using the pint library to handle units for us: https://pint.readthedocs.io/en/0.9/

Reply:
(or at least just leave a comment with the unit)